config/output_schemas/intram_by_pg_uk_schema.toml

```diff
@@ -0,0 +1,11 @@
+["Detailed product groups (Alphabetical product groups A-AH)"]
```

Comment:
I'm not sure how the schemas operate. Assuming that's its one to one with outputs, I've left two separate ones in. This also allows for room for change if the schemas do ever differ.

Up to you on this one since you have more knowledge of how they are used.

Reply:
Yeah, just checked and specific schemas are needed for exporting from DAP.  I'll show you that tomorrow :)